port cypress test to playwright and remove cypress

vinaybadgujar102 opened the pull request:
resolves #1879

tijlleenders reviewed (changes requested):
Nice!
Did you do a search for all occurences of 'cypress' to remove everything?

I think it is mentioned in the Readme.md

vinaybadgujar102 commented:
@tijlleenders, I previously searched for and removed all occurrences of Cypress, but it seems one instance in the README was missed. I have now updated it in the latest commit.

tijlleenders reviewed (approved):
Great!

@Tushar-4781 code OK?
